src/tools/rbd_mirror/InstanceReplayer.cc

```diff
@@ -370,12 +370,9 @@ void InstanceReplayer<I>::queue_start_image_replayers() {
 }
 
 template <typename I>
-void InstanceReplayer<I>::start_image_replayers(int r) {
-  dout(10) << dendl;
-
-  std::lock_guard locker{m_lock};
+void InstanceReplayer<I>::start_image_replayers(
+  const std::unique_lock<ceph::mutex>&) {
```

Comment:
Nit: Don't indent the  parameter the same is the first line of code:
```suggestion
    const std::unique_lock<ceph::mutex>&) {
```